platform/openide.util.enumerations/nbproject/project.properties

```diff
@@ -15,7 +15,7 @@
 # specific language governing permissions and limitations
 # under the License.
 
-javac.source=1.4
+javac.source=1.6
```

Comment:
This is not going to work. The target must be `1.4` as the source code is using `enum` in the package name. Of course such compilation can succeed only on older JDKs. The `build.xml` has already been modified to skip the compilation on JDK11 and newer.